sdk/purview/purview-sharing-rest/recordings/browsers/received_shares_operations/recording_attach_received_share.json

```diff
@@ -1,7 +1,7 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://accountname.purview.azure.com/share/receivedShares/530232e8-0fc5-4138-8e39-f856b42773b7?api-version=2023-02-15-preview",
+      "RequestUri": "https://accountname.purview.azure.com/share/receivedShares/206016dd-fd49-420c-9545-9663badda4e3?api-version=2023-05-30-preview",
       "RequestMethod": "PUT",
```

Comment:
Currently the suggested position for recording files are assent repo, could you follow the guide to migrate these files?

https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/ASSET_SYNC_MIGRATION.md

Reply:
We requested access and the request was denied with the following comment:
_**Please join a specific team https://aka.ms/azsdk/permission**_

Could you point me to a different link where we can get the required permission? @yamanwahsheh

Reply:
You can try this: https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/49/Request-Access-to-Azure-SDK-repos

Reply:
A follow up question to help clarify, which permissions should we be requesting out of this list? Can you send me a specific one?

Reply:
Should be migrated now.